lib/jhove_service.rb

```diff
@@ -46,13 +46,15 @@ def run_jhove(content_dir, fileset_file=nil)
   # @param fileset_file [Pathname,String] the pathname of the file listing which files should be processed.  If nil, process all files.
   # @return [String] The jhove-toolkit command to be exectuted in a system call
   def get_jhove_command(content_dir, fileset_file=nil)
+    raise "Folder #{content_dir} not found" unless File.directory? content_dir
+    args = "-h xml -o #{jhove_output} "
     if fileset_file.nil?
-      args = "edu.stanford.sulair.jhove.JhoveCommandLine #{content_dir.to_s}"
```

Comment:
confirmed w/ @peetucket that this change brings the jhove command invocation inline with what's suggested in the current jhove documentation.